CHANGELOG.md

```diff
@@ -11,9 +11,13 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Added
 
+- We added support for selecting and using CSL Styles in JabRef's OpenOffice/LibreOffice integration for inserting bibliographic and in-text citations into a document.
```

Comment:
You added this sentence twice.

Reply:
Hmm seems like Oliver and I wrote the same changelog entry (mistakenly duplicated because I had missed the issue number).

Reply:
Resolved